maven-compat/src/test/java/org/apache/maven/repository/legacy/StringWagon.java

```diff
@@ -72,14 +73,7 @@ public void fillInputData( InputData inputData )
             resource.setContentLength( content.length() );
             resource.setLastModified( System.currentTimeMillis() );
 
-            try
-            {
-                inputData.setInputStream( new ByteArrayInputStream( content.getBytes( "UTF-8" ) ) );
-            }
-            catch ( UnsupportedEncodingException e )
-            {
-                throw new Error( "broken JVM", e );
-            }
+            inputData.setInputStream( new ByteArrayInputStream( content.getBytes(StandardCharsets.UTF_8) ) );
```

Comment:
Formatting

Reply:
That's strange, I've fixed it after running `clean verify`

Will fix a bit later

Reply:
Actually checkstyle doesn't think current formatting violates something, but I've fixed what you said.